it/uk/gov/hmrc/apigatekeeperapprovalsfrontend/connectors/SubmissionConnectorISpec.scala

```diff
@@ -58,6 +58,7 @@ class SubmissionConnectorISpec extends BaseConnectorIntegrationISpec with GuiceO
     val markSubmission = markedSubmission
     val requestedBy    = "bob@blockbusters.com"
     val reason         = "reason"
+    val escalatedTo    = "Mr Edmund Blackadder"
```

Comment:
Is this wise? Feels like a weak escalation point :smile:

Reply:
Lol